HiP-DataStore/Core/WriteModel/TagIndex.cs

```diff
@@ -0,0 +1,44 @@
+﻿using PaderbornUniversity.SILab.Hip.DataStore.Model.Events;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace PaderbornUniversity.SILab.Hip.DataStore.Core.WriteModel
+{
+    public class TagIndex : IDomainIndex
+    {
+        private readonly Dictionary<int, TagInfo> _tag = new Dictionary<int, TagInfo>();
```

Comment:
My fault. "ev.Id" is of course a tag ID, not an event ID, so everything is fine. Sorry!